src/libraries/Microsoft.PowerFx.Core/Errors/ErrorUtils.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Globalization;
+using System.Text;
+using Microsoft.PowerFx.Core.Localization;
+using Microsoft.PowerFx.Core.Utils;
+
+namespace Microsoft.PowerFx.Core.Errors
+{
+    internal class ErrorUtils
+    {
+        /// <summary>
+        /// Format error messages replacing the placeholder to argument values.
+        /// </summary>
+        /// <param name="message">Error message to format.</param>
+        /// <param name="locale">CultureInfo information.</param>
+        /// <param name="args">Original error message args.</param>
+        /// <returns></returns>
+        internal static string FormatMessage(string message, CultureInfo locale, params object[] args)
+        {
+            if (message == null)
+            {
+                return null;
+            }
+
+            var sb = new StringBuilder();
```

Comment:
>var sb = new StringBuilder(); [](http://example.com/codeflow?start=12&length=29)

We don't need to use a StringBuilder since we only append one thing to it

Reply:
To be addressed in #812